lib/Queue.rb

```diff
@@ -1,22 +1,39 @@
 class Queue
+attr_accessor :store
```

Comment:
We do not want to do this. This makes @store available outside of the Queue class which means that someone could do something like:

```
q = Queue.new
# ...stuff here
q.store[2] = 100
```

So basically it undoes the limitations that we put on accessing/modifying elements in a Queue by letting you directly modify the Queue.

To solve this, just remove this line `attr_accessor :store`
We similarly wouldn't want it to be `attr_reader` either